sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -781,14 +779,14 @@
           {
             "name": "id",
             "in": "path",
-            "description": "The ID of the device whose device twin is to be replaced",
+            "description": "The unique identifier of the device whose twin is to be replaced.",
```

Comment:
Is it not enough to say "The unique identifier of the device"?

Reply:
same as above